src/Query/Admin/TelemetryQuery/DbalTelemetryQuery.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Buddy\Repman\Query\Admin\TelemetryQuery;
+
+use Buddy\Repman\Entity\Organization\Member;
+use Buddy\Repman\Query\Admin\TelemetryQuery;
+use Buddy\Repman\Service\Telemetry\Entry\Organization;
+use Buddy\Repman\Service\Telemetry\Entry\Package;
+use Doctrine\DBAL\Connection;
+
+final class DbalTelemetryQuery implements TelemetryQuery
+{
+    private Connection $connection;
+
+    public function __construct(Connection $connection)
+    {
+        $this->connection = $connection;
+    }
+
+    public function usersCount(): int
+    {
+        return (int) $this
+            ->connection
+            ->fetchColumn(
+                'SELECT COUNT(id) FROM "user"',
+            );
+    }
+
+    /**
+     * @return Organization[]
+     */
+    public function organizations(int $limit = 100, int $offset = 0): array
+    {
+        return array_map(function (array $data): Organization {
+            return new Organization(
+                $data['id'],
+                $data['tokens'],
+                $data['has_anonymous_access'],
+                $data['members'],
+                $data['owners'],
+            );
+        }, $this->connection->fetchAll(
+            'SELECT
+                o.id,
+                COUNT(t.value) tokens,
+                o.has_anonymous_access,
+                COUNT(m.id) members,
+                COUNT(ow.id) owners
+            FROM "organization" o
+            LEFT JOIN "organization_token" t ON t.organization_id = o.id
+            LEFT JOIN "organization_member" m ON m.organization_id = o.id AND m.role = :role_member
+            LEFT JOIN "organization_member" ow ON ow.organization_id = o.id AND ow.role = :role_owner
```

Comment:
This double join can be avoided with `FILTER` :sunglasses: 
```sql
COUNT(m.id) FILTER (WHERE m.role = :role_member) AS members,
COUNT(m.id) FILTER (WHERE m.role = :role_owner) AS owners
```
check this please, I'm not sure :wink: